db/migrations/versions/e86cbf275e45_.py

```diff
@@ -5,36 +5,54 @@
 Create Date: 2023-08-18 12:47:57.470108
 
 """
-from alembic import op
 import sqlalchemy as sa
+from alembic import op
 from sqlalchemy.dialects import postgresql
 
 # revision identifiers, used by Alembic.
-revision = 'e86cbf275e45'
-down_revision = 'f3332786bb2e'
+revision = "e86cbf275e45"
+down_revision = "f3332786bb2e"
 branch_labels = None
 depends_on = None
 
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.create_table('feedback',
-    sa.Column('id', postgresql.UUID(as_uuid=True), nullable=False),
-    sa.Column('application_id', postgresql.UUID(as_uuid=True), nullable=False),
-    sa.Column('fund_id', sa.String(), nullable=False),
-    sa.Column('round_id', sa.String(), nullable=False),
-    sa.Column('section_id', sa.String(), nullable=False),
-    sa.Column('feedback_json', sa.JSON(), nullable=False),
-    sa.Column('status', sa.Enum('NOT_STARTED', 'IN_PROGRESS', 'SUBMITTED', 'COMPLETED', name='status'), nullable=False),
-    sa.Column('date_submitted', sa.DateTime(), nullable=True),
-    sa.ForeignKeyConstraint(['application_id'], ['applications.id'], name=op.f('fk_feedback_application_id_applications')),
-    sa.PrimaryKeyConstraint('id', name=op.f('pk_feedback')),
-    sa.UniqueConstraint('id', 'section_id', name=op.f('uq_feedback_id'))
+    op.create_table(
+        "feedback",
+        sa.Column("id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column("application_id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column("fund_id", sa.String(), nullable=False),
+        sa.Column("round_id", sa.String(), nullable=False),
+        sa.Column("section_id", sa.String(), nullable=False),
+        sa.Column("feedback_json", sa.JSON(), nullable=False),
+        sa.Column(
+            "status",
+            postgresql.ENUM(
+                "NOT_STARTED",
+                "IN_PROGRESS",
+                "SUBMITTED",
+                "COMPLETED",
+                name="status",
+                create_type=False,
+            ),
+            nullable=False,
+        ),
```

Comment:
Most of this is formatting changes, however this enum was changed to use `create_type=False` as it already exists.

You'll need to downgrade before migration @AbuNavsa:


```bash
flask db downgrade f3332786bb2e
flask db upgrade
```